extensions/ql-vscode/src/databases/local-databases-ui.ts

```diff
@@ -60,7 +60,7 @@ enum SortOrder {
   DateAddedDesc = "DateAddedDesc",
 }
 
-type LanguageFilter = QueryLanguage | "All";
+type LanguageFilter = QueryLanguage | "all";
```

Comment:
cc @starcke, since this probably affects your WIP branch 😬 

I renamed `All" -> "all" to make everything consistently lowercase.

Reply:
Ignore me! I reverted that commit, to avoid messy conflicts. We can change it later, if we care enough 😄